src/main/java/org/icgc/argo/program_service/grpc/ProgramServiceImpl.java

```diff
@@ -99,6 +105,8 @@ public void createProgram(CreateProgramRequest request, StreamObserver<CreatePro
   @Override
   public void getProgram(GetProgramRequest request, StreamObserver<GetProgramResponse> responseObserver) {
     val shortName = request.getShortName().getValue();
+    authorizationService.requireProgramUser(shortName);
```

Comment:
According to the Endpoint Authorization Matrix here: https://wiki.oicr.on.ca/pages/viewpage.action?pageId=110627486, `getProgram` should use `requireProgramAdmin`.

Reply:
You can get YOUR programs; but not programs that you have no access to.